libraries/aws_iam_groups.rb

```diff
@@ -11,13 +11,15 @@ class AwsIamGroups < AwsResourceBase
     end
   '
 
-  attr_reader :table
+  attr_reader :table, :inline_policy_names
```

Comment:
this is not required. The FilterTable registration takes care of creating the method .

Reply:
Tested and confirmed this line is not required. Thanks for the catch.